### ISSUE TITLE
Can you give out the loss function of YOLOv3？

### ISSUE THREAD

**Issue by Fighting-JJ** [YunYang1994/tensorflow-yolov3#18]
Reading the code  about loss_layer() without loss function is just hard to understand. 
Thanks a lot ! @YunYang1994

**Comment by YunYang1994** [comment]
Hope it helps you https://stats.stackexchange.com/questions/380012/yolov3-loss-function

**Comment by muye5** [comment]
> Hope it helps you https://stats.stackexchange.com/questions/380012/yolov3-loss-function

this implementation of yolov3 loss is divided by box number, I am not sure whether it's good or not.
for my understanding, the loss of darknet version is just sum over all boxes in one image. any advantage is there for counting only one box?
